src/odemis/gui/xmlh/generate_xrc.py

```diff
@@ -169,12 +174,19 @@ def GenerateInitResourcesEmbedded(self, resourceFilename, resourceDocument):
 
 
 def main(args=None):
-    if not args:
-        args = sys.argv[1:]
-
     inputFiles = []
-    for arg in args:
-        inputFiles += glob.glob(arg)
+
+    if args:
+        outputFilename = os.path.splitext(args[0])[0] + "_xrc.py"
+        for arg in args:
+            inputFiles += glob.glob(arg)
```

Comment:
So this will keep working for `src/odemis/gui/win/dialog.xrc` and other xrc files, right?

Reply:
Correct! it works, see below
```
patel@laptop:~/development/odemis/src/odemis/gui/xmlh$ ./generate_xrc.py ../win/dialog.xrc 
Resources written to ../win/dialog_xrc.py.
```